stripe/_stripe_client.py

```diff
@@ -19,10 +19,16 @@
     new_http_client_async_fallback,
 )
 from stripe._api_version import _ApiVersion
+from stripe._stripe_object import StripeObject
+from stripe._stripe_response import StripeResponse
 from stripe._webhook import Webhook, WebhookSignature
 from stripe._event import Event
 
-from typing import Optional, Union, cast
+from typing import Any, Dict, Optional, Union, cast
+
+from stripe._raw_request import raw_request as _raw_request  # noqa
+from stripe._raw_request import raw_request_async as _raw_request_async  # noqa
+from stripe._raw_request import deserialize as _deserialize  # noqa
```

Comment:
The above 3 methods use the global api requestor. I believe one of the key differentiators of the StripeClient model is that there is a dedicated api requestor per StripeClient that gets used instead of the global one - that is how we get to set different api-key/account per StripeClient. 

Therefore, I think we need a bigger refactor.

Reply:
I think this is done.  I removed _raw_request.py and merged those methods directly into StripeClient so I could use its _requestor member, and updated the tests to match.